resources/zib/zib-SOAPReport.xml

```diff
@@ -105,7 +105,7 @@
       <path value="Composition.section" />
       <sliceName value="soapLine" />
       <short value="SOAPLine" />
-      <definition value="Container of the SOAPLine concept.This container contains all data elements of the SOAPLine concept." />
+      <definition value="Container of the SOAPLine concept. This container contains all data elements of the SOAPLine concept." />

```

Comment:
Is de spelfout ook doorgegeven aan het zibcentrum?

Reply:
Ik heb deze fout nu gemeld in ZIB-1920.